apps/teams/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+const { resolve } = require('node:path');
+
+const project = resolve(__dirname, './tsconfig.json');
+
+module.exports = {
+  extends: ['@elba-security/eslint-config-custom/next', 'plugin:@typescript-eslint/recommended'],
```

Comment:
```suggestion
  extends: ['@elba-security/eslint-config-custom/next'],
```
It's better to stick to the configuration we are providing for now. If in your opinion it's lake of rules or some might be changed, feels free to open an issue or discute it with me in pm.